src/interaction/inspector.jl

```diff
@@ -830,7 +839,8 @@ function show_data(inspector::DataInspector, plot::BarPlot, idx)
                 linewidth = a.indicator_linewidth, linestyle = a.indicator_linestyle,
                 visible = a.indicator_visible, inspectable = false
             )
-            translate!(p, Vec3f(0, 0, a.depth[]))
+            wf_depth = a.depth[] * 0.999 # put the wireframe below the tooltip
+            translate!(p, Vec3f(0, 0, wf_depth))
```

Comment:
This feels like a bit of hack, but it works. Is there a different standard for ensuring z ordering?

Reply:
Even with this depth fix, the wire mesh sometimes bleeds into the triangle part of the tooltip. Only when it's drawn to the right, but not when it's drawn to the left, see:
![Screenshot 2024-08-05 at 11 51 52 AM](https://github.com/user-attachments/assets/dd4cb83a-3691-4f1b-aa43-3f71802519d9)
![Screenshot 2024-08-05 at 11 52 05 AM](https://github.com/user-attachments/assets/09b43bcd-e2bf-4058-a492-946f09c2b7e9)

Reply:
Actually I think this should be removed and replaced by a depth_shift like a couple other indicators have. That way it's independent on camera details.

Reply:
Hmm, I'm not sure what you have in mind. I've tried a few variations with surprising results.

This snippet solves the problem, but completely ignores the specified `a.depth` attribute:
```
            p = wireframe!(
                scene, bbox, model = model, color = a.indicator_color,
                linewidth = a.indicator_linewidth, linestyle = a.indicator_linestyle,
                visible = a.indicator_visible, inspectable = false
            )
            push!(inspector.temp_plots, p)
```

This snippet uses the depth attribute and a depth shift value but also doesn't solve the problem. Probably because the translate call overwrites the depth shift value?
```
            p = wireframe!(
                scene, bbox, model = model, color = a.indicator_color,
                linewidth = a.indicator_linewidth, linestyle = a.indicator_linestyle,
                visible = a.indicator_visible, inspectable = false,
                depth_shift = -1f-3,
            )
            translate!(p, Vec3f(0, 0, a.depth[]))
            push!(inspector.temp_plots, p)
```

Reply:
Just doing
```
p = wireframe!(scene, bbox, ..., depth_shift = -1f-3)
```
with no translate should place the wireframe in front of bbox. In fact you shouldn't even need to set depth_shift since wireframe has it at <0 by default. 
depth_shift acts after model (i.e. also translate!()) and camera transformation when everything is normalized to a -1..1 depth range. It's meant to resolve issues where two things are plotted at the same depth, but one should be in front.

The reason why triangle doesn't behave right is probably the missing `o[3]` here: 
https://github.com/MakieOrg/Makie.jl/blob/3977fac9184237aa21bfba953ca2a51094d4dbec/src/basic_recipes/tooltip.jl#L179
I thought I fixed that already but I guess not.

Reply:
Thank you for the suggestions. I just pushed both changes.